Jumpit/JumpitAPI.swift

```diff
@@ -0,0 +1,75 @@
+//
+//  JumpitAPI.swift
+//  Jumpit
+//
+//  Created by 이지훈 on 5/22/24.
+//
+
+import Foundation
+import Moya
+
+enum JumpitAPI {
+    case postResume(data: ResumeData)
+    case fetchMyResume(userId: String)
+    case toggleResumePrivacy(resumeId: String)
+    case searchPositions(keyword: String)
+    case searchPositionsFiltered(keyword: String, categories: [Int])
+    case fetchPositionDetail(positionId: String)
+}
+
+extension JumpitAPI: TargetType {
+    var baseURL: URL {
+        guard let urlString = Bundle.main.object(forInfoDictionaryKey: "BaseURL") as? String,
+              let url = URL(string: urlString) else {
+            fatalError("BaseURL not set in plist for this environment")
+        }
+        return url
+    }
+
+    var path: String {
+        switch self {
+        case .postResume:
+            return "/resumes"
+        case .fetchMyResume(let userId):
+            return "/resumes/\(userId)"
+        case .toggleResumePrivacy(let resumeId):
+            return "/resumes/\(resumeId)"
+        case .searchPositions, .searchPositionsFiltered:
+            return "/positions"
+        case .fetchPositionDetail(let positionId):
+            return "/positions/\(positionId)"
+        }
+    }
+
```

Comment:
```suggestion
extension JumpitAPI: JumpitAPIEndpoint {
    var path: String {
        switch self {
        case .postResume:
            return "/resumes"
        case .fetchMyResume(let userId):
            return "/resumes/\(userId)"
        case .toggleResumePrivacy(let resumeId):
            return "/resumes/\(resumeId)"
        case .searchPositions, .searchPositionsFiltered:
            return "/positions"
        case .fetchPositionDetail(let positionId):
            return "/positions/\(positionId)"
        }
    }

```
<3번 방식> JumpitBaseApi에서 엔드포인트를 지정해주셨기 때문에 Moya 라이브러리를 사용할 때 baseURL과 path 속성이 자동으로 결합되어 최종 URL을 형성하게 됩니다! 따라서 다시 baseUrl을 써주시지 않아도 될 것 같아요!